android/app/project.properties

```diff
@@ -0,0 +1,14 @@
+# This file is automatically generated by Android Tools.
+# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
+#
+# This file must be checked in Version Control Systems.
+#
+# To customize properties used by the Ant build system edit
+# "ant.properties", and override values to adapt the script to your
+# project structure.
+#
+# To enable ProGuard to shrink and obfuscate your code, uncomment this (available properties: sdk.dir, user.home):
+#proguard.config=${sdk.dir}/tools/proguard/proguard-android.txt:proguard-project.txt
+
+# Project target.
+target=android-21
```

Comment:
That's what Lint tool does in IDEs, it parses the minSdkVersion and makes sure the code can run from that version and beyond.
Besides that if you want to use newer Android features that will appear only in new versions, you could not if you use old build target.

There is the official recommendation here:
http://developer.android.com/tools/projects/projects-eclipse.html
"Select a Compile With API version. This setting specifies what version of the SDK to compile your project against. We strongly recommend using the most recent version of the API."